.buildkite/pipeline.yml

```diff
@@ -461,60 +461,87 @@ steps:
+          --dt 1secs --t_end 100secs
+          --dt_save_to_disk 10secs
+        artifact_paths: "diagnostic_edmfx_test_box/*"
         agents:
           slurm_mem: 20GB
 
-      - label: ":genie: EDMFX Bomex on a sphere"
-        command:
-          - julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_bomex_sphere --ode_algo SSP33ShuOsher --apply_limiter false --initial_condition Bomex --moist equil --turbconv edmfx --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --surface_setup Bomex --z_elem 60 --z_stretch false --z_max 3e3 --dt 0.001secs --t_end 0.1secs --dt_save_to_disk 0.01secs --reference_job_id edmf_bomex
-        artifact_paths: "edmfx_bomex_sphere/*"
+      - label: ":genie: Diagnostic EDMFX Bomex in a box"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl --job_id diagnostic_edmfx_bomex_box
+          --initial_condition Bomex --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --surface_setup Bomex
+          --turbconv diagnostic_edmfx --edmfx_entr_detr true
+          --moist equil
+          --config box --hyperdiff true --kappa_4 1e8
+          --x_max 1e5 --y_max 1e5 --x_elem 2 --y_elem 2 --z_elem 60 --z_max 3e3 --z_stretch false
+          --dt 10secs --t_end 1hours --dt_save_to_disk 5mins
+        artifact_paths: "diagnostic_edmfx_bomex_box/*"
         soft_fail: true
         agents:
           slurm_mem: 20GB
 
       - label: ":genie: EDMFX advection test in a box"
         command: >
           julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_adv_test_box
-          --initial_condition MoistAdiabaticProfileEDMFX --edmfx_adv_test true --turbconv edmfx --edmfx_entr_detr false
+          --initial_condition MoistAdiabaticProfileEDMFX
+          --turbconv edmfx --edmfx_adv_test true
+          --moist equil
           --config box --hyperdiff true --kappa_4 1e8
           --x_max 1e4 --y_max 1e4 --x_elem 2 --y_elem 2 --z_elem 45 --z_max 30e3 --dz_bottom 30
-          --moist equil --ode_algo SSP33ShuOsher --apply_limiter false --dt 10secs --t_end 1000secs
+          --ode_algo SSP33ShuOsher --apply_limiter false --dt 10secs --t_end 1000secs
           --dt_save_to_disk 100secs
-
         artifact_paths: "edmfx_adv_test_box/*"
         agents:
           slurm_mem: 20GB
 
+      - label: ":genie: EDMFX Bomex in a box"
+        command: >
+          julia --color=yes --project=examples examples/hybrid/driver.jl --job_id edmfx_bomex_box
+          --initial_condition Bomex --subsidence Bomex --edmf_coriolis Bomex --ls_adv Bomex --surface_setup Bomex
+          --turbconv edmfx --edmfx_entr_detr true --edmfx_sgs_flux true
+          --moist equil
+          --config box --hyperdiff true --kappa_4 1e8
+          --x_max 1e5 --y_max 1e5 --x_elem 2 --y_elem 2 --z_elem 60 --z_max 3e3 --z_stretch false
+          --dt 10secs --t_end 1hours --dt_save_to_disk 5mins
```

Comment:
dt = 10secs may be too large if you have, say, updraft velocity > 5m/s and dz = 50m here.